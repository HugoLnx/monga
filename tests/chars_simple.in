int main() {
	int num;
	char a, b, c;
	int num2;
	num = 300;
	a = 'A';
	b = 'B';
	c = 'C';
	num2 = 350;
	printf("num1: %d\n", num);
	printf("a: %c\n", a);
	printf("b: %c\n", b);
	printf("c: %c\n", c);
	printf("num2: %d\n", num2);
}
