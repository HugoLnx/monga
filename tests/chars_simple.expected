num1: 300
a: A
b: B
c: C
num2: 350
